articles/specs/launch.0.1.1.xsd

```diff
@@ -608,5 +609,6 @@
           </xs:documentation>
         </xs:annotation>
       </xs:attribute>
+    </xs:complexType>
```

Comment:
It's not really clear to me that this is the right thing to do here.  I think the "complexType" here is indeed the ref to "env", "param", and "remap", and it should end there.  The rest of the attributes seem to be attributes on "node", so I think this should be moved back where it was, and the indentation should be fixed.  Or maybe I'm misunderstanding?

Reply:
Same as in https://github.com/ros2/design/pull/278#discussion_r409651844.